src/App.vue

```diff
@@ -109,18 +118,28 @@ export default {
   },
   methods: {
     ...mapActions('elementalChat', ['setChannelPolling', 'updateProfile']),
-    ...mapActions('holochain', ['skipBackoff']),
+    ...mapActions('holochain', ['skipBackoff', 'holoLogout']),
     ...mapMutations(['setAgentHandle', 'setErrorMessage']),
+    ...mapMutations('elementalChat', ['setNeedsHandle']),
     agentHandleEntered () {
       if (this.internalAgentHandle === '') return
       this.updateProfile(this.internalAgentHandle)
       this.dialog = false
     },
+    closeModal () {
+      this.setNeedsHandle(false)
+    },
     clearErrorMessage () {
       this.setErrorMessage('')
     },
     retryNow () {
       this.skipBackoff()
+    },
+    disconnectedHoloLogout () {
+      log('refreshed page to lose display of data')
```

Comment:
what does this mean? "lose display of data"

Reply:
It means 'clear page data'. I didn't want to say removed page data... as the ec-ui localstorage data is still present.  I'll updated to say clear.